docs/source/best-practices/checkpointing.rst

```diff
@@ -2,21 +2,21 @@
 Checkpoint/Restart Discovery Jobs
 *****************************************
 
-The complexity of HPC systems may introduce unpredictable behaviors and may result in job failures due to hardware or software. Applying fault tolerance techniques to your HPC workflows allows your jobs to become more resilient to crashes, partition time limits and hardware failures.
+The complexity of HPC systems may introduce unpredictable behaviors and result in job failures due to hardware or software. Applying fault tolerance techniques to your HPC workflows makes your jobs more resilient to crashes, partition time limits, and hardware failures.
```

Comment:
Since we don't discuss any other "fault tolerance techniques" in this page, can we remove, "Applying fault tolerance techniques to your HPC workflows . . ."  And maybe just take the checkpointing definition from the next subsection and move that up to this paragraph?